oeps/architectural-decisions/oep-0026-arch-realtime-events.rst

```diff
@@ -138,7 +139,10 @@ For details on integrating with Caliper, please see the :ref:`caliper_realtime_e
 Anonymized User ID
 ==================
 
-The *LMS user_id* will be used to uniquely identify a user in the Open edX system. This decision is detailed in :ref:`oep-32`.
+Users will be identified to external systems using a UUID that is associated uniquely with a single user and the external system type with which the UUID can be shared.  This decision overrides :ref:`oep-32`. and is captured in `ADR 0001-externalid.rst`_
```

Comment:
1. The period is in the wrong place in the final sentence.
2. Do you want to create an issue in https://github.com/openedx/open-edx-proposals/issues about OEP-32 and what you think needs to happen? Or a PR that adds a warning to OEP-32 about its questionable state? (I admit that I think something should be done, but am unsure of exactly what to do and am not thinking in this area right now, so this will probably just sit if you decide to do nothing.)

Reply:
I've created a basic ticket to capture the work.  I think that OEP-32 should be marked as obsolete.

I'm happy to do that if you are happy to act at the arbiter for that small change.  I'm not planning to take on the substantial rewrite at this time.

Reply:
Maybe we could point to the issue as the Superseded link, and clarify that the link should be updated once the issue is complete?

Reply:
Do you mean in OEP-32 or here?  I think that as an initial useful step we would update OEP-32 to indicate that it was obsolete and link to the ADR for reference.